pulp_ansible/app/serializers.py

```diff
@@ -132,7 +133,18 @@ def validate(self, data):
         data = super().validate(data)
 
         if data.get("requirements_file"):
-            parse_collections_requirements_file(data["requirements_file"])
+            parsed_file = parse_collections_requirements_file(data["requirements_file"])
+            urls_to_validate = [data["url"]]
+            for name, version, source in parsed_file:
+                if source:
+                    urls_to_validate.append(source)
+
+            for url in urls_to_validate:
+                result = re.findall(r"/v(\d)/collections", url)
+                if not result:
+                    raise serializers.ValidationError(
+                        _("Invalid URL {url}".format(url=data["url"]))
```

Comment:
I couldn't think a better error message

Reply:
Maybe `Inavlid URL {url}. Ensure the URL ends in either '/v2/collections/' or '/v3/collections/'.`